### ISSUE TITLE
on_joinplayer callback crash

### ISSUE THREAD

**Issue by BuckarooBanzay** [pandorabox-io/pandorabox.io#509]
```
2020-05-16 15:44:01: ERROR[Main]: ServerError: AsyncErr: ServerThread::run Lua: Runtime error from mod 'default' in callback on_joinplayer(): ...share/minetest/games/minetest_game/mods/default/init.lua:23: attempt to index local 'info' (a nil value)
2020-05-16 15:44:01: ERROR[Main]: stack traceback:
2020-05-16 15:44:01: ERROR[Main]: 	...share/minetest/games/minetest_game/mods/default/init.lua:23: in function <...share/minetest/games/minetest_game/mods/default/init.lua:16>
2020-05-16 15:44:01: ERROR[Main]: 	/usr/local/share/minetest/builtin/game/register.lua:429: in function </usr/local/share/minetest/builtin/game/register.lua:413>

```

**Comment by OgelGames** [comment]
Happened again...
```
2020-07-03 13:53:18: ERROR[Main]: ServerError: AsyncErr: ServerThread::run Lua: Runtime error from mod 'default' in callback on_joinplayer(): ...share/minetest/games/minetest_game/mods/default/init.lua:23: attempt to index local 'info' (a nil value)
2020-07-03 13:53:18: ERROR[Main]: stack traceback:
2020-07-03 13:53:18: ERROR[Main]: 	...share/minetest/games/minetest_game/mods/default/init.lua:23: in function 'func'
2020-07-03 13:53:18: ERROR[Main]: 	...ocal/share/minetest/builtin/profiler/instrumentation.lua:106: in function <...ocal/share/minetest/builtin/profiler/instrumentation.lua:100>
2020-07-03 13:53:18: ERROR[Main]: 	/usr/local/share/minetest/builtin/game/register.lua:429: in function </usr/local/share/minetest/builtin/game/register.lua:413>
```

**Comment by BuckarooBanzay** [comment]
yeah, i saw :disappointed:

**Comment by BuckarooBanzay** [comment]
Relevant code in `minetest_game`:

https://github.com/minetest/minetest_game/blob/dcb3df0a38be9599b32538f24e383e5d6ef81606/mods/default/init.lua#L16-L33

```lua
minetest.register_on_joinplayer(function(player)
	-- Set formspec prepend
	local formspec = [[
			bgcolor[#080808BB;true]
			listcolors[#00000069;#5A5A5A;#141318;#30434C;#FFF] ]]
	local name = player:get_player_name()
	local info = minetest.get_player_information(name)
	if info.formspec_version > 1 then
		formspec = formspec .. "background9[5,5;1,1;gui_formbg.png;true;10]"
	else
		formspec = formspec .. "background[5,5;1,1;gui_formbg.png;true]"
	end
	player:set_formspec_prepend(formspec)

	-- Set hotbar textures
	player:hud_set_hotbar_image("gui_hotbar.png")
	player:hud_set_hotbar_selected_image("gui_hotbar_selected.png")
end)
```

Cause: `minetest.get_player_information()` can return nil (somehow? :shrug:)

Fixed in 5.3.0: https://github.com/minetest/minetest/commit/ac368af4fe27f61f5a4209cdfe90956ff745993c
